connectors/src/connectors/microsoft/temporal/mime_types.ts

```diff
@@ -4,6 +4,7 @@ export function getMimeTypesToSync({ pdfEnabled }: { pdfEnabled: boolean }) {
     // TODO(pr): support those
     // "text/plain",
     "application/vnd.openxmlformats-officedocument.presentationml.presentation",
+    "application/vnd.ms-excel", // equivalent to "text/csv"
```

Comment:
if it's equivalent, let's use the same mimetype in gdrive and microsoft

Reply:
I mean that's the Microsoft type for "text/csv"

Reply:
Oh ok! could you change the com to say exactly that? `Microsoft type for "text/csv"`

Reply:
✅ 😸